doc/security/specs/secure_boot/index.md

```diff
@@ -32,7 +32,7 @@ firmware upgrade support.
 ## RISC-V Concepts:
 
 *   **PMP:** Physical Memory Protection unit from the
-    [RISC-V ISA Volume II: Privileged Architecture](rv-isa-priv) specification.
```

Comment:
`[rv-isa-priv]` with https://riscv.org/specifications/privileged-isa/ is not valid anymore. Could you update this to https://riscv.org/technical/specifications/ or https://github.com/riscv/riscv-isa-manual/releases/latest